dashboard/config/blocks/GamelabJr/gamelab_location_variable_set.json

```diff
@@ -0,0 +1,19 @@
+{
+  "category": "Location",
+  "config": {
+    "simpleValue": true,
+    "name": "location_variable_set",
+    "blockText": "set {VAR} to {VAL}",
+    "args": [
+      {
+        "name": "VAR",
+        "assignment": true,
```

Comment:
Nit: Tempting to make this a separate field instead of part of "args", but it makes sense that these get populated from the block inputs, in order.  If we could start over, maybe these would be called "fields" instead of "args"?